Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -461,12 +461,21 @@ open class Kommunicate: NSObject,Localizable, KMPreChatFormViewControllerDelegat
                 if isPreChatEnable {
                     UserDefaults.standard.set(appSetting.chatWidget?.preChatGreetingMsg!, forKey: "leadCollectionTitle")
                     leadArray = appSetting.leadCollection!
+                    //test purpose
+                    let option = [LeadCollectionDropDownField(value: "1"),
```

Comment:
For testing purpose, add it in ViewController, where the customer will be adding. They can't change this file.